main.go

```diff
@@ -286,9 +288,13 @@ func generateMethods(path openAPI.PathItem) (methods []PathMethod) {
 
 func generateResponses(responses *openAPI.Responses) (orderedResponses []MethodResponse) {
 	for status, response := range responses.StatusCodeResponses {
+
+		json, _ := json.MarshalIndent(response.ResponseProps.Schema, "", "  ")
```

Comment:
Do you need to check for the potentially error here? No sure what we would need to do if this does fail but worth considering.

Reply:
Thanks; makes sense to check for an error in the marshal process. Have updated to include an `err` variable and if statement that calls `log.Fatal` to reflect the way errors are handled elsewhere in the file